lib/linter/apply-disable-directives.js

```diff
@@ -87,6 +157,10 @@ function applyDirectives(options) {
     const unusedDisableDirectives = options.directives
         .filter(directive => directive.type === "disable" && !usedDisableDirectives.has(directive))
         .map(directive => ({
+            fix: {
+                range: getDirectiveRemovalRange(directive, options.sourceCode),
+                text: ""
+            },
```

Comment:
When we are removing individual rules, we could accidentally remove all rules but leave the directive (that will then disallow all rules):

```js
/* eslint no-undef: 2, no-proto: 2, no-iterator: 2 */

// eslint-disable-next-line no-proto,, no-iterator
undef_foo();
``` 

autofixed to:

```js
/* eslint no-undef: 2, no-proto: 2, no-iterator: 2 */

// eslint-disable-next-line 
undef_foo();

```

This doesn't happen with 1 comma (`no-proto, no-iterator`) because the ranges are overlapping in that comma (one fix wants to remove `no-proto,`, the other `, no-iterator`), so only one fix gets applies in the first pass, while the second pass removes the whole comment because at that point there is only one rule in the directive.

The `,,` example is an edge case, but relying on the overlapping of successive ranges seems error-prone. Can we instead replace the whole comment with a new version that doesn't contain the rule?

Fox example, instead of a fix on a smaller range:

`no-proto,` -> ` `

the error for `no-proto` would produce a fix on the range of the whole comment:

`// eslint-disable-next-line no-proto,, no-iterator` -> `// eslint-disable-next-line , no-iterator`